src/components/_common/office-details/phonePoster/AudienceChannels.tsx

```diff
@@ -0,0 +1,57 @@
+import { BodyShort } from '@navikt/ds-react';
+import { LenkeBase } from 'components/_common/lenke/LenkeBase';
+import { AudienceContact } from 'types/content-props/office-details-props';
+import { parsePhoneNumber } from '../utils';
+
+import styles from './AudienceChannels.module.scss';
+
+type AudienceChannelsProps = {
+    publikumskanaler: AudienceContact[];
+};
+
+export const AudienceChannels = ({
+    publikumskanaler,
+}: AudienceChannelsProps) => {
+    const buildHrefWithPrefix = (href: string) => {
+        const prefix = href.includes('@') ? 'mailto:' : 'tel:';
+        return `${prefix}${href}`;
+    };
+
+    const buildLinkText = (href: string) => {
+        return href.includes('@') ? href : parsePhoneNumber(href);
+    };
+
+    const createLink = (href: string) => {
+        return (
+            <LenkeBase href={buildHrefWithPrefix(href)}>
+                {buildLinkText(href)}
+            </LenkeBase>
+        );
+    };
+
+    const buildChannel = (channel: AudienceContact) => {
+        return (
+            <BodyShort
+                key={channel.sortOrder}
+                className={styles.audienceChannels}
+            >
+                {`${channel.beskrivelse}: `}
+                {createLink(channel.epost || channel.telefon)}
+            </BodyShort>
+        );
+    };
```

Comment:
Disse epost/telefon feltene, har norg noe validering på disse? Er det garantert at en publikumskanal har en og bare en av epost eller telefon satt, og at denne er gyldig?

Isåfall, kan vi skrive om disse funksjonene til å behandle href'en som telefon eller epost avhengig av hvilket felt som er satt, slik at vi ikke trenger å sjekke for '@'.

Dersom det _ikke_ er validering i norg på disse, så bør vi kanskje ha vår egen validering her. Hva om f.eks. epost-feltet er satt til en string uten '@', denne bør da helst ikke behandles som et telefonnummer.

Reply:
Ja, NORG har validering av både e-post, telefonnr og en ugyldig kombinasjon dersom man forsøker å legge inn begge. Jeg har tatt en ny runde og forenklet slik at vi bare sjekker på om hver av de finnes. Siden NORG uansett validerer at det kun finnes én av de så foreslår jeg at vi ikke gjør noen ytterligere sjekk enn at innholdet ikke er nullish?

<img width="1128" alt="Screenshot 2023-04-04 at 10 32 31" src="https://user-images.githubusercontent.com/1443997/229734946-2ad9abc5-e209-4b8c-a8c3-d141274b96f0.png">